crates/aptos/src/move_tool/mod.rs

```diff
@@ -1649,6 +1700,10 @@ async fn is_staging_area_empty(txn_options: &TransactionOptions) -> CliTypedResu
 pub struct ClearStagingArea {
     #[clap(flatten)]
     pub(crate) txn_options: TransactionOptions,
+
+    /// Address of the `large_packages` move module for chunked publishing
```

Comment:
I would add another line that you only have to override when not on testnet or mainnet